### ISSUE TITLE
New 'surround_enabled' option giving error

### ISSUE THREAD

**Issue by rleeden** [avantrec/soco-cli#25]
Hi,

I've just upgraded to version 0.4.26 and testing new 'sub_enabled' and 'surround_enabled'. The first of these works fine, but the second errors. I'm expecting both to give the answer as 'on'. Output from both commands with log set to info:

sonos --log info -l Lounge sub_enabled

```
2021-12-14 10:42:34,784 sonos.py:130 - main() - Setting up handlers for: [<Signals.SIGINT: 2>, <Signals.SIGTERM: 15>]
2021-12-14 10:42:34,784 sonos.py:237 - main() - No 'SPKR' environment variable set
2021-12-14 10:42:34,785 sonos.py:260 - main() - Found 1 action sequence(s): [['Lounge', 'sub_enabled']]
2021-12-14 10:42:34,785 utils.py:582 - get_speaker() - Using local speaker list
2021-12-14 10:42:34,785 match_speaker_names.py:18 - speaker_name_matches() - Found exact speaker name match for 'Lounge'
2021-12-14 10:42:34,786 sonos.py:470 - main() - Invoking 'run_command' with '<SoCo object at ip 192.168.0.233> sub_enabled ...'
2021-12-14 10:42:34,858 api.py:125 - run_command() - Return value: (0, 'on', '')
on
```

sonos --log info -l Lounge surround_enable`
```
2021-12-14 10:44:38,413 sonos.py:130 - main() - Setting up handlers for: [<Signals.SIGINT: 2>, <Signals.SIGTERM: 15>]
2021-12-14 10:44:38,413 sonos.py:237 - main() - No 'SPKR' environment variable set
2021-12-14 10:44:38,413 sonos.py:260 - main() - Found 1 action sequence(s): [['Lounge', 'surround_enabled']]
2021-12-14 10:44:38,414 utils.py:582 - get_speaker() - Using local speaker list
2021-12-14 10:44:38,414 match_speaker_names.py:18 - speaker_name_matches() - Found exact speaker name match for 'Lounge'
2021-12-14 10:44:38,415 sonos.py:470 - main() - Invoking 'run_command' with '<SoCo object at ip 192.168.0.233> surround_enabled ...'
2021-12-14 10:44:38,415 api.py:88 - run_command() - Exception: 'tuple' object has no attribute 'switch_to_coordinator'
2021-12-14 10:44:38,415 api.py:125 - run_command() - Return value: (1, '', "Error: 'tuple' object has no attribute 'switch_to_coordinator'")
Error: 'tuple' object has no attribute 'switch_to_coordinator'
```
sonos --version
```
soco-cli version: 0.4.26
soco version:     0.25.0
python version:   3.9.1
```

Thanks,
Richard

**Comment by pwt** [comment]
Thanks for the bug report -- I'll take a look later.

Could you describe the Sonos devices involved (or provide the output of `sonos -l Lounge sysinfo`? Thanks.

**Comment by rleeden** [comment]
sonos -l Lounge sysinfo

```
Report generated on: 2021-12-14 10:55 UTC (Tuesday)

Zone Name    IP Address     Visible    CoOrd    CoOrd IP        Vol.   Mute    State    Model Name    Model No.    HW Version     SW Version
-----------  -------------  ---------  -------  -------------  ------  ------  -------  ------------  -----------  -------------  -------------------
Bathroom     192.168.0.72   Yes        No       192.168.0.69     14    On      STOPPED  Play:1        S1           1.8.3.7-2.0    66.4-23300 (13.4.1)
Bedroom      192.168.0.69   Yes        Yes                       6     On      STOPPED  One SL        S22          1.28.1.6-2.1   66.4-23300 (13.4.1)
Kitchen      192.168.0.182  Yes        No       192.168.0.70     18    Off     PLAYING  Move          S17          1.25.1.8-2.10  66.4-23300 (13.4.1)
Kitchen      192.168.0.77   No         No       192.168.0.70     18    Off     PLAYING  Move          S17          1.25.1.8-2.10  66.4-23300 (13.4.1)
Lounge       192.168.0.141  No         No       192.168.0.233    10    On      Bonded   Sub           S26          1.32.1.6-2.1   66.4-23300 (13.4.1)
Lounge       192.168.0.198  No         No       192.168.0.233    10    On      Bonded   One           S18          1.26.1.9-2.1   66.4-23300 (13.4.1)
Lounge       192.168.0.233  Yes        Yes                       10    On      STOPPED  Arc           S19          1.27.1.8-2.2   66.4-23300 (13.4.1)
Lounge       192.168.0.248  No         No       192.168.0.233    10    On      Bonded   One           S18          1.26.1.9-2.1   66.4-23300 (13.4.1)
Office       192.168.0.151  No         No       192.168.0.70     15    Off     PLAYING  Play:1        S12          1.20.1.6-2.2   66.4-23300 (13.4.1)
Office       192.168.0.70   Yes        Yes                       15    Off     PLAYING  Play:1        S12          1.20.1.6-2.2   66.4-23300 (13.4.1)

Sonos model numbers present: S1, S12, S17, S18, S19, S22, S26.
Device counts: 10 total Sonos device(s), 7 unique model(s).
```

**Comment by pwt** [comment]
Thanks for spotting this. Should be fixed in v0.4.27 (just released).
